src/journal-remote/journal-gatewayd.c

```diff
@@ -1040,7 +1040,7 @@ int main(int argc, char *argv[]) {
                 int flags =
                         MHD_USE_DEBUG |
                         MHD_USE_DUAL_STACK |
-                        MHD_USE_PIPE_FOR_SHUTDOWN |
+                        MHD_USE_ITC |
```

Comment:
I think we should update that [comment](https://github.com/systemd/systemd/pull/5166/commits/39dfd35fe1a49948d1d9404a50cb1d0552cd1adf#diff-2d7e61049e387b011a9df20244a7922dL1031) too
> We force MHD_USE_PIPE_FOR_SHUTDOWN here ...